app/src/main/res/layout/dialog_calculate_calories.xml

```diff
@@ -7,7 +7,7 @@
     <EditText
         android:id="@+id/edit_text_weight"
         android:layout_width="0dp"
-        android:layout_height="@dimen/button_height_short"
+        android:layout_height= "48dp"
```

Comment:
If I am not wrong, there is a @dimen/button_height_medium. Please check this and use this if possible so that there are no hardcoded values

Reply:
yep! Thanks for the suggestion.I have made the required changes.